src/Settings/Features.php

```diff
@@ -96,7 +99,10 @@ public static function all_features(): array {
 			NewDesignCountrySelector::class,
 			NewDesignNavigationBar::class,
 			GoogleSheetReplacesSmartsheet::class,
-			ListingPagePagination::class,
+			NewListingPagesTags::class,
```

Comment:
I added a toggle per type, because we need to adjust the caching to work for all the types.